docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,119 @@
+        - Change `ConnectionConfiguration` in the `config.json` generated [here](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/terraform_loadtest.md#copy-default-config-1)
+        - Configure `InstanceConfiguration` in the same `config.json` (which as mentioned earlier, creates the seeds mm-server's database with required data for the loadtest). Note that a heavier config with `NumPosts` would take a very long time get seeded. Please refer to the [NB](/#nb) section below to manually seed the database from a backup, in order to bypass 'data-generation.
+    
+    - Start the load test with `go run ./cmd/ltctl loadtest start`
+        - Once a loadtest is running, its status can be checked with `go run ./cmd/ltctl loadtest status`.
+        - `ssh` into one of the agent machines, and `cat ~/mattermost-load-test-ng/logs/ltagent.log` to verify the load-test is working without errors.
+        - Open the Grafana deployment with the URL from `go run ./cmd/ltctl deployment info`. 
+        - It takes some time for the deployment to stabilize, i.e while the loadtest tool connects `MaxActiveUsers` count of users to app, there might be a big count of HTTP 4xx errors in this duration.
+            
+            A sample error count vs time graph: ![error-vs-time](https://i.imgur.com/RSH1Szl.png)
+    
+    - "My load test is running, now what?"
+        - If it's a [bounded](https://github.com/mattermost/mattermost-load-test-ng/blob/497554e376ef23d548947bf331c8bdce6ce453d6/docs/faq.md#what-is-a-bounded-load-test) loadtest, it has to be manually stopped with `go run ./cmd/ltctl loadtest stop` [after an hour](https://community.mattermost.com/core/pl/45woi49ru7yrj8r8upzaqhog3a)
+        - If it's an [unbounded](https://github.com/mattermost/mattermost-load-test-ng/blob/497554e376ef23d548947bf331c8bdce6ce453d6/docs/faq.md#what-is-an-unbounded-load-test) loadtest, the load-test will finish with a stdout listing the maximum concurrent users the deployment supports. The load-test status check command will say status as `Done` when it's complete.
+
+#### Analysing load-test results
+
+"My load tests ran successfully, what to make of it?"
+    - In case of unbounded load-tests, when they finish, `go run ./cmd/ltctl loadtest status` would give you a count of maximum concurrent users which is a metric to compare the performance of that version of mattermost-server.
+    - In case of two bounded loadtests with same `MaxConcurrentUsers` count, one can [generate a report](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/compare.md) comparing performance of various server metrics. 
+    <!-- devfix: should we allow comparision of specific store and api functions? -->
+    - In case of both bounded and unbounded loadtests, one can create grafana dashboards to analyze performances of the new features by filtering api metrics relevant to the new handler. Here's an example.
```

Comment:
```suggestion
    - In case of both bounded and unbounded loadtests, one can create Grafana dashboards to analyze the performance of the new features by filtering API metrics to only what's relevant to the new handler. Here's an example.
```